application/ui/components/input/child-custom-object.jsx

```diff
@@ -0,0 +1,22 @@
+/** @jsx React.DOM */
+/* jshint global console */
+'use strict';
+
+var _               = require('underscore');
+var React           = require('react');
+var ObjectParam;
+
+module.exports = React.createClass({
+    displayName : 'ChildObjectParameter',
+    componentWillMount: function() {
+        ObjectParam = require('./custom-object');
+    },
+    render : function()
```

Comment:
To follow the (unspoken) convention, let's add line breaks above `render` and `componentWillMount`